contract/models/account_analytic_contract.py

```diff
@@ -26,6 +25,12 @@ class AccountAnalyticContract(models.Model):
         comodel_name="res.partner",
         string="Partner (always False)",
     )
+    contract_type = fields.Selection(
+        selection=[
+            ('sale', _('Sale')),
```

Comment:
Selection fields don't require to translate the label this way (they are extracted automatically), so remove `_` calls